_schemaV2.graphql

```diff
@@ -9423,6 +9481,12 @@ type Mutation {
     # Parameters for SubmitPendingOffer
     input: CommerceSubmitPendingOfferInput!
   ): CommerceSubmitPendingOfferPayload
+
+  # Update value of impulse_conversation_id on order
+  commerceUpdateImpulseConversationId(
```

Comment:
Looks like this is the result of https://github.com/artsy/exchange/pull/1054. @lidimayra do you have context if this mutation will be or is being used? I assume without it in the schema, clients can't actually use it.